addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -149,7 +149,8 @@ class ComposerTextInput extends Component {
         this._textareaRef.el.value = this.composer.textInputContent;
         this._textareaRef.el.setSelectionRange(
             this.composer.textInputCursorStart,
-            this.composer.textInputCursorEnd
+            this.composer.textInputCursorEnd,
+            this._textareaRef.el.selectionDirection
```

Comment:
As we discussed the other day, I think this should be saved in the models with the other values and restored from the models too.

This only works by luck, but if OWL is destroying and remounting the component this will not work.

Reply:
Done